src/qt/addresstablemodel.cpp

```diff
@@ -85,7 +86,7 @@ class AddressTablePriv
             BOOST_FOREACH(const PAIRTYPE(CTxDestination, CAddressBookData)& item, wallet->mapAddressBook)
             {
                 const CBitcoinAddress& address = item.first;
-                bool fMine = IsMine(*wallet, address.Get());
+                bool fMine = wallet->IsMine(address.Get());
```

Comment:
You need to change this , as you have modified the is mine function to include a block ie CBlockindex